hangman/lib/game/assist.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Hangman.Assist do
+  @moduledoc """
+  Implementation of Hangman assist functions
+  """
+
+  def split_word(word) do
+   String.graphemes(word)
+  end
+  
+  
+  #invalid input
+  def guess(_, :false, _, _, _, _) do
+    raise ArgumentError, message: "Please enter a valid input!"
+  end
+
+  #zero turns
+  # have to put full word into letters at the end or 
+  # will not show up in human_player (only for loss funcs)
+
+  def guess(game, :true, _, _, _, 0) do
+    %{ game | game_state: :lost }
+    |> Map.put(:letters, String.split(game.word))
+  end
+
+  #one turn and word is wrong
+  def guess(game, :true, _, _, :false, 1) do
+    %{ game | game_state: :lost }
+    |> Map.put(:letters, String.split(game.word))  
+  end
+
+  #repeat letter
+  def guess(game, :true, _, :true, _, _) do
```

Comment:
## -4 bug

The spec says this should return `:already_used` and not `:letter_repeat`